### ISSUE TITLE
Need help or maybe a bug

### ISSUE THREAD

**Issue by medienopfer** [laradic/support#1]
We tried to install a new version of our Laravel System which includes blade-extenions and got this error now... Is there a problem with 5.3 update in your package?
# 

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - radic/blade-extensions 3.0.2 requires laradic/support ^1.2.5 -> no matching package found.
    - radic/blade-extensions 3.0.1 requires laradic/support ^1.2.5 -> no matching package found.
    - radic/blade-extensions 3.0.0 requires laradic/support 1.2.\* -> no matching package found.
    - Installation request for radic/blade-extensions 3.0.\* -> satisfiable by radic/blade-extensions[3.0.0, 3.0.1, 3.0.2].

Potential causes:
- A typo in the package name
- The package is not available in a stable-enough version according to your minimum-stability setting
  see https://getcomposer.org/doc/04-schema.md#minimum-stability for more details.

**Comment by RobinRadic** [comment]
Right. This is something i have failed to anticipate. Here's the situation:
- I have moved the organisation `laradic` to `laradic-old`.
- I have created the organistation `laradic` again.
- I have created everything from scratch. All repos, branches etc.

I never thought that blade-extensions v3 was relying on that  repository. [which can be found here](https://github.com/laradic-old/support)

I could release a patch update for v3. But it'll have to be from PR. Currently working on blade-extensions 7.0.0. Which supports 5.0, 5.1, 5.2 and 5.3. So i don't feel like putting time into the matter
